src/wp-admin/includes/ajax-actions.php

```diff
@@ -188,7 +188,11 @@ function wp_ajax_wp_compression_test() {
 	}
 
 	if ( ini_get( 'zlib.output_compression' ) || 'ob_gzhandler' === ini_get( 'output_handler' ) ) {
-		update_site_option( 'can_compress_scripts', 0 );
+		if ( is_multisite() ) {
+			update_site_option( 'can_compress_scripts', 0 );
+		} else {
+			update_option( 'can_compress_scripts', 0 );
+		}
```

Comment:
This change is because `update_site_option()` does not allow control over the `$autoload` parameter when calling `update_option()`, is that right?

I think it would be better to explicitly pass "yes" here so that it's clear why the change is here. For someone that doesn't have the context, that wouldn't be obvious. Adding a comment in addition would be even better.
```suggestion
		// Use `update_option()` on single site to mark the option for autoloading.
		if ( is_multisite() ) {
			update_site_option( 'can_compress_scripts', 0 );
		} else {
			update_option( 'can_compress_scripts', 0, 'yes' );
		}
```